openraft/Cargo.toml

```diff
@@ -55,7 +55,7 @@ bench = []
 
 # Enable backtrace when generating an error.
 # Stable rust does not support backtrace.
-bt  = ["anyerror/backtrace", "anyhow/backtrace"]
+bt  = ["anyhow/backtrace"]
```

Comment:
The backtrace feature in `anyerror` is still needed to be turned on.

Reply:
anyerror needs an upgrade to make it work on rust 1.63.

Reply:
I fired an updated release of anyerror to adapt changes introduced by stable-1.63 and the latest nightly:
https://crates.io/crates/anyerror/0.1.7